provider/k8s_test.go

```diff
@@ -398,3 +396,29 @@ func Test_parquetIteratorFromReader(t *testing.T) {
 		index += 1
 	}
 }
+
+func Test_parquetNoRowsIteratorFromReader(t *testing.T) {
```

Comment:
Also test get num rows returning the correct error here

Reply:
Done